### ISSUE TITLE
Hover on column is not removed when leaving the tree node

### ISSUE THREAD

**Issue by Andries-Smit** [mendix/TreeViewAndGridView#19]
Hover class is only removed, when the cursor is moved to the next item. 
https://github.com/mendix/TreeViewAndGridView/blob/master/src/TreeView/widget/TreeView.js#L654

![image](https://cloud.githubusercontent.com/assets/6724749/15391993/aed03d06-1dc4-11e6-8a39-21d9bf0d8e4d.png)

Expect the hove to be removed whenever the move is not above the item.

Maybe it should be removed on the "mouseleave" event, but I am not sure if this will break the dnd

**Comment by Andries-Smit** [comment]
Adding a live connect did not do the trick.. : (
```
          lc(this, this.treeNode, "mouseleave", {
                "gg_column_wrapper": function (target, e) {
                    dojo.removeClass(this.hoveredCol, "gg_col_hover");
                    return true;
                }
            });
```

**Comment by JelteMX** [comment]
Fix from #20 added, thanks! Closing issue and creating a new release today
